### ISSUE TITLE
"#byebye has no method 'masonry'"

### ISSUE THREAD

**Issue by ghost** [desandro/masonry#156]
Trying to figure out the basics of masonry with a tumblr theme, and for some reason I can't call masonry with my id class.  It continually tells me that "#byebye" (the id in which all the tumblr posts are stored) has no method "masonry." I thought it was because I didn't include the necessary files...but I have:

```
<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js"></script>
<script type="text/javascript" src="http://static.tumblr.com/wnngsb4/F1elwoq63/jquery.masonry.min.js"></script>
<script type="text/javascript">
$(window).load(function(){
    ('#byebye').masonry();
});
</script>
```

I'm at a loss...don't really know why I can't call the masonry function because I've seen it referenced this way in all the examples with no issues.

_Edit:_ Okay so I'm a dolt. Forgot a beginning declaration with the javascript, so it didn't want to accept anything. Issue resolved.
